content/blog/2024-06-27-using-bitnami-valkey-chart/index.md

```diff
@@ -0,0 +1,182 @@
++++
+title="Using Bitnami's Valkey chart"
+date=2024-06-27 01:01:01
+description="An introductory example using Bitnami's Valkey chart"
+aliases= [
+    "/blog/2024/06/using-bitnami-valkey-chart/"
+]
+authors=["rafariossaa"]
++++
+
+Valkey is a high-performance key/value datastore that supports workloads such as caching, and message queues, supporting many data types including strings, numbers, hashes, bitmaps, and more. Valkey can run in standalone or cluster mode for replication and high availability.
+
+This blog will serve as walk-through on how you can deploy and use the [Bitnami Helm chart for Valkey](https://github.com/bitnami/charts/tree/main/bitnami/valkey).
+
+# Assumptions and prerequisites
+
+Before starting the deployment, make sure that you have the following prerequisites:
+- An operational Kubernetes cluster.
+- An installed and configured kubectl CLI and Helm v3.x package manager. If you need help with these steps, check our article “[Learn how to install kubectl and Helm v3.x.](https://docs.bitnami.com/kubernetes/get-started-kubernetes#step-3-install-kubectl-command-line)”
+- Optional: Access to [VMware Tanzu Application Catalog](https://app-catalog.vmware.com/catalog).
+
+# Deploying the Bitnami package for the Valkey Helm chart
```

Comment:
A little explanation about the two methods would be useful, maybe say something like "you can do this with Bitnami and it's fully open source", then when you switch into talking about Tanzu "If you're a Tanzu user, you can also do it this way". 

It should be really clear that you're using a service for Tanzu's App Catalog.

Reply:
I think this point is mentioned in the paragraph a few lines below: 
```
You can deploy the the [open source](https://github.com/bitnami/charts/blob/main/LICENSE.md) Bitnami-packaged Valkey Helm chart from the Bitnami Application Catalog. Alternatively, if you have access to an enterprise Tanzu Application Catalog instance, it can also be deployed from there.
```